packages/vue/src/components/confirm-reset-password.vue

```diff
@@ -15,8 +14,7 @@ import BaseFormFields from './primitives/base-form-fields.vue';
 
 const { state, send } = useAuth();
 
-const useAuthShared = createSharedComposable(useAuthenticator);
```

Comment:
This is actually a wrong usage of `createSharedComposable`. 

`createSharedComposable` creates a closure with shared state value when it is called. Previously, each auth subcomponents called its own `createdSharedComposable`, which creates its own closure, and ended up not sharing states.

https://vueuse.org/shared/createSharedComposable/

Reply:
Thanks for digging in here!